pack/tsk/tskp.json

```diff
@@ -2579,13 +2579,14 @@
         "code": "09107",
         "cost": null,
         "deck_limit": 2,
+        "errata_date": "2025-07-11",
         "faction_code": "survivor",
         "illustrator": "David Hovey",
         "name": "Predator or Prey",
         "pack_code": "tskp",
         "position": 107,
         "quantity": 2,
-        "text": "Max 2 [[Dilemma]] per round.\n<b>Revelation</b> - If there are no enemies in play, draw 1 card. Otherwise, you must decide (choose one):\n- Each unengaged enemy moves once toward the nearest investigator.\n- Each investigator disengages from each enemy engaged with them and moves once away from the nearest enemy.",
+        "text": "Max 2 [[Dilemma]] per round.\n<b>Revelation</b> - If there are no enemies in play, draw 1 card. Otherwise, you must decide (choose one):\n- Each unengaged non-[[Elite]] enemy moves once toward the nearest investigator.\n- Each investigator disengages from each enemy engaged with them and moves once away from the nearest enemy.",
```

Comment:
Just a note that though the errata document itself says Predator or Prey is card 20, it's not. Need some errata errata!